middleman_protocol/RELEASE-VERSION

```diff
@@ -0,0 +1 @@
+0.8.0+dev194.gce34fd0
```

Comment:
We do not want to commit this file. Please remove it and add it to `.gitignore`.

In development you don't need it because `setup.py` can still import `version`.
In deployment we are going to generate it during the build.